### ISSUE TITLE
importing Breakpoint

### ISSUE THREAD

**Issue by LegendT** [hagenburger/livingstyleguide#116]
When trying to compile livingstyleguide with a file with import 'breakpoint' it complains that it can't find the import.

Using version 2.0.0 alpha.3

**Comment by hagenburger** [comment]
Could you please provide the paths for all files, the command to execute, and from which part it has been executed? The import refers to a Sass file I guess (not to a LSG file)? Thanks!

**Comment by LegendT** [comment]
sass/style.scss file contains:

// gem imports
@import "compass";
@import "breakpoint";
@import "susy";

// base
@import "base/variables";
@import "base/normalize";

// helpers.
@import "helpers/mixins/mixins";
@import "helpers/placeholders/placeholders";
@import "helpers/mixins/accessibility";
@import "helpers/mixins/images";


@import "base/scaffolding";

// templates
@import "templates/Global/Global";

__________________________________________________

styleguide.lsg file contains
source: "sass/style.scss"
title: "My Nice & Beautiful Living Style Guide"

---------------------------------------------------------------------------------

command executed is

livingstyleguide compile styleguide.lsg styleguide.html

**Comment by hagenburger** [comment]
I would try to add a [Gemfile](http://bundler.io/gemfile.html) and additionally list all gems there:

``` ruby
source "https://rubygems.org"

gem "livingstyleguide", "2.0.0.alpha.3"
gem "compass"
gem "breakpoint"
gem "susy"
```

Then run (first two lines only once):

```
gem install bundler 
bundle
bundle exec livingstyleguide compile styleguide.lsg styleguide.html
```

----

This is an issue which is planned to be solved more easily in v2.0.0 but not done yet. Let me know, if the work around works for you.

**Comment by hagenburger** [comment]
As I haven’t hear anything, I guess this is solved for you. I’m closing this.
